assets/section-featured-blog.css

```diff
@@ -1,12 +1,13 @@
 .blog-placeholder {
-  margin: 0 1.5rem;
   background: rgb(var(--color-background));
+  width: 100%;
+  color: inherit;
+  position: relative;
```

Comment:
I can't tell what these 2 are doing. Can you confirm we still need them and show me what they're doing?